docs/src/meshes/p4est_mesh.md

```diff
@@ -386,6 +386,7 @@ transfinite map of the straight sided hexahedral element to find
 
 Also for a mesh in standard Abaqus format there are no qualitative changes when going from 2D to 3D.
 The most notable difference is that boundaries are formed in 3D by faces defined by four nodes while in 2D boundaries are edges consisting of two elements.
+Note that standard Abaqus also defines quadratic element types. In Trixi.jl, these higher-order elements are currently only supported in 2D, i.e., 8-node quadrilaterals.
```

Comment:
Are *only* 8-node quads available? Or does this PR implement the infrastructure such that higher order with more nodes per element are (in theory) supported?

Reply:
So at the moment yes, see lines **1931 to 1933** in `p4est_mesh.jl`. This should be easily generalizable, but I am actually not sure if there are elements with order higher than quadratic defined, see e.g. http://130.149.89.49:2080/v2016/books/usb/default.htm?startat=pt06ch28s01ael02.html

Reply:
Yeah, that seems to be the case. The ABAQUS format can only natively handle up to quadratic boundaries.